powerhome/playbook
# Play-122 Collapsible kit add curser pointer on hover

![React Kit Coverage](https://img.shields.io/badge/React%20Kit%20Coverage-13.39%25-red)
Before:
![Fullscreen_5_4_22__10_43_AM](https://user-images.githubusercontent.com/84344530/166748746-c92a0421-b037-463a-aa64-db60be59d007.jpg)

After
![Fullscreen_5_4_22__1_07_PM](https://user-images.githubusercontent.com/84344530/166971682-f78cc121-ce23-4321-b64b-707f9b35f543.jpg)
![Fullscreen_5_4_22__1_08_PM](https://user-images.githubusercontent.com/84344530/166971684-4a4165f7-56cd-47e7-b964-cc65475bf778.jpg)


#### Breaking Changes

No, Changes are simply to the kit itself ascetically in order to show when something is clickable not if its clickable. 

#### Runway Ticket URL

[[INSERT URL]](https://nitro.powerhrg.com/runway/backlog_items/PLAY-122)

#### How to test this

Need to check at Kit in use and see if the point changes when hovering over the clickable area of the Collapse

#### Checklist:
